### ISSUE TITLE
downdetector.com: privacy

### ISSUE THREAD

**Issue by jordansworld** [uBlockOrigin/uAssets#25966]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [X] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://downdetector.com/status/discord/
```

### Description

tracker: `https://downdetector.com/cdn-cgi/trace`

### Other extensions used

n/a

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml


```
</details>

**Comment by ScottBeeson** [comment]
Is there any chance this broke all the charts on downdetector.com? They are suddenly being blocked by uBO and never were previously.

**Comment by mapx-** [comment]
example of broken page ?

**Comment by jordansworld** [comment]
i apologize looks like its a bad filter:
https://www.reddit.com/r/uBlockOrigin/s/EiJMf0X6jX

**Comment by jimmysofat6864** [comment]
I pressed the open preferences button and it shows the comments but then scrolling no no longer works. 

![Image](https://github.com/user-attachments/assets/93211560-cd3d-4deb-9a07-0b71a6eb7ccb)

**Comment by jordansworld** [comment]
> I pressed the open preferences button and it shows the comments but then scrolling no no longer works.
> 
> ![Image](https://github.com/user-attachments/assets/93211560-cd3d-4deb-9a07-0b71a6eb7ccb)

same here. i submitted a new issue to adguard

**Comment by gorhill** [comment]
> I pressed the open preferences button and it shows the comments but then scrolling no no longer works.

What preferences button? Which exact URL can this preferences button be found?

**Comment by jimmysofat6864** [comment]
Pick any website on the main page and scroll down to the comments section.
There the comments shouldn’t be visible and the open preferences box should
be visible. Selecting open preferences causes the page to show the comments
but break scrolling.



A sample link is
https://downdetector.com/status/doordash/

**Comment by jimmysofat6864** [comment]
The website I used in my screenshot was https://downdetector.com/status/deepseek/?v=2025.

**Comment by stephenhawk8054** [comment]
Can you open new tab > reproduce the issue and on that page > click on uBO icon > &#128172; [Report button](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > scroll to the bottom > click on `Troubleshooting Information` and copy paste it here?

**Comment by jimmysofat6864** [comment]
> Can you open new tab > reproduce the issue and on that page > click on uBO icon > 💬 [Report button](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > scroll to the bottom > click on `Troubleshooting Information` and copy paste it here?

This is what I get. 

```
uBlock Origin: 1.62.0
Chromium: 133
filterset (summary):
 network: 313048
 cosmetic: 215523
 scriptlet: 59428
 html: 0
listset (total-discarded, last-updated):
 added:
  adguard-generic: 104588-5827, 7h.12m
  adguard-mobile: 10652-88, 7h.12m
  adguard-spyware-url: 1994-137, 7h.12m
  adguard-spyware: 145802-29878, 7h.12m
  block-lan: 71-0, 7h.12m
  curben-phishing: 1651-0, 8h.33m
  adguard-cookies: 33457-99, 7h.12m
  ublock-cookies-adguard: 3412-18, 7h.12m
  fanboy-cookiemonster: 44625-3716, 7h.12m
  ublock-cookies-easylist: 3412-3412, 7h.12m
  adguard-popup-overlays: 28790-2400, 7h.12m
  adguard-mobile-app-banners: 5903-1137, 7h.12m
  [6 lists not shown]: [too many]
 default:
  user-filters: 1-0, never
  ublock-filters: 40333-568, now Δ
  ublock-badware: 12016-37, now Δ
  ublock-privacy: 2272-39, now Δ
  ublock-unbreak: 2561-14, now Δ
  ublock-quick-fixes: 389-1, now Δ
  easylist: 69785-10, now Δ
  easyprivacy: 53382-663, now Δ
  urlhaus-1: 20400-0, 8h.33m
  plowe-0: 3469-445, 7h.12m
filterset (user): [array of 1 redacted]
trustedset:
 added: [array of 3 redacted]
userSettings:
 cloudStorageEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 702 ms (selfie)
 maxAssetCacheWait: 88 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 12
 network:
  downdetector.com: 2
  cookielaw.org: 3
  disqus.com: 3
  googletagmanager.com: 1
  liadm.com: 1
  twitter.com: 2
 extended:
  ##.ad
  ##.app-promo-card:not(#style_important)
  ##.ad.leaderboard
  ###ad-leaderboard
  ###zdcFloatingBtn
  ###adchoicesBtn
  ###onetrust-consent-sdk
  ##.onetrust-pc-dark-filter
  ##+js(set-constant, zdconsent.consentGiven, true)
  ##html:style(overflow:auto!important)
  ##html[style*="overflow"]:style(overflow: auto !important;)
  ##+js(trusted-set-cookie, d_prefs, MjoxLGNvbnNlbnRfdmVyc2lvbjoyL…
  ##+js(set-constant, Navigator.prototype.globalPrivacyControl, fa…
  ##+js(set-constant, navigator.globalPrivacyControl, false)
  ##+js(trusted-replace-xhr-response, '/,"expanded_url":"([^"]+)",…
  ##+js(set-cookie, twtr_pixel_opt_in, N)
  ##+js(json-prune-xhr-response, data.home.home_timeline_urt.instr…
  ##+js(json-prune-xhr-response, data.search_by_raw_query.search_t…
  ##+js(json-prune-xhr-response, data.threaded_conversation_with_i…
  ##+js(json-prune-xhr-response, data.user.result.timeline_v2.time…
  ##+js(json-prune-xhr-response, data.immersiveMedia.timeline.inst…
  #@#+js(trusted-replace-xhr-response, '/,"expanded_url":"([^"]+)"…
  #@#+js(json-prune-xhr-response, data.home.home_timeline_urt.inst…
  #@#+js(json-prune-xhr-response, data.search_by_raw_query.search_…
  #@#+js(json-prune-xhr-response, data.threaded_conversation_with_…
  #@#+js(json-prune-xhr-response, data.user.result.timeline_v2.tim…
  #@#+js(json-prune-xhr-response, data.immersiveMedia.timeline.ins…
  ###placement-top
  ###email-signup
  ##.comment-policy-refresh
  ###thread-subscribe-button
```

**Comment by jimmysofat6864** [comment]
These are also the filters I am using. 

![Image](https://github.com/user-attachments/assets/6ac8c984-50a2-4a0e-aaf3-29c670bc43d8)

**Comment by stephenhawk8054** [comment]
Test again in a new browser profile with only uBO and its default settings.

**Comment by jimmysofat6864** [comment]
> Test again in a new browser profile with only uBO and its default settings.

The cookies box appears and I can select the options and once I select accept the box closes and I can scroll as normal.

**Comment by stephenhawk8054** [comment]
On your old profile, check which cookie list is causing issue for you.

**Comment by mapx-** [comment]
AdGuard – Cookie Notices issue

```adb
downdetector.com#@##onetrust-consent-sdk
```

**Comment by Yuki2718** [comment]
Looks fixed by https://github.com/AdguardTeam/AdguardFilters/commit/7a52616106197072a5a95367d31eba2c2ebdaeb6

**Comment by gorhill** [comment]
> Looks fixed by https://github.com/AdguardTeam/AdguardFilters/commit/7a52616106197072a5a95367d31eba2c2ebdaeb6

`trusted-set-cookie` from an AdGuard list won't work in uBO though.
